src/plot_topoplotseries.jl

```diff
@@ -9,21 +9,24 @@ Multiple miniature topoplots in regular distances.
     `Figure`, `GridLayout`, `GridPosition`, or GridLayoutBase.GridSubposition to draw the plot.
 - `data::Union{<:Observable{<:DataFrame},DataFrame}`\\
     DataFrame with data or Observable DataFrame. Requires a `time` column. 
-- `Δbin::Real`\\
-    A number for how large one time bin should be.\\
-    `Δbin` is in units of the `data.time` column.\\
-    Should be `0` if `mapping.col` or `mapping.row` are categorical.
 
 ## Keyword arguments (kwargs)
+- `bin_width::Real = nothing`\\
```

Comment:
Before the function didn't require a time column (as in the line above). This is somewhat confusing, you could use mapping.x to define what the "time" column is, e.g. what should be binned.

Not sure how to put that into words, but this is still possible right?

Eg. Napping.x= :continuosA, bi width =1?

Reply:
I am not sure I completley get it. Should I write it in docstrings or in configs

Reply:
definitely change in docs if we can still do mapping.x = :bla  instead of the default mapping.x = :time